lib/castle/middleware/sensor.rb

```diff
@@ -103,7 +101,13 @@ def app_id_js_tag(env)
       def identify_js_tag(env)
```

Comment:
Merge `app_id_js_tag` and `identify_js_tag` and call it something like `config_js_tag`

Reply:
Yeah agree, I also want to restructure this so they end up in the same script tag